tailwind.config.ts

```diff
@@ -20,12 +20,14 @@ const config = {
     extend: {
       colors: {
         border: "hsl(var(--border))",
+        "card-border": "#FAFAFA",
```

Comment:
use variables in hsl, see `globals.css`

Reply:
Sorry but I did not add the hsl styles that is how the dev branch is the only changes I made are the rgb (cardText and card-border)

Reply:
fixed